src/view/configuration/configuration.jsx

```diff
@@ -75,23 +77,25 @@ const getInitialValues = initInfo => {
       });
     });
   } else {
-    instances = [createDefaultInstance()];
+    instances = [createDefaultInstance(initInfo)];
   }
 
   return {
     instances
   };
 };
 
-const getSettings = values => {
+const getSettings = ({ values, initInfo }) => {
+  const instanceDefaults = getInstanceDefaults(initInfo);
   return {
     instances: values.instances.map(instance => {
       const trimmedInstance = {
-        name: instance.name,
-        propertyId: instance.propertyId
+        name: instance.name
```

Comment:
`name` is the only property that needs to be saved to the settings object when it matches the form default value (`alloy`). We may even want to remove the default form value of "alloy" and just have the user always provide a name...I'm not sure. Considering "alloy" is the default in the base code in our alloy documentation, I think it probably makes sense to keep it as a default here as well.